components/formats-bsd/src/loci/formats/out/PyramidOMETiffWriter.java

```diff
@@ -167,6 +167,7 @@ public void close() throws IOException {
           saver.overwriteIFDOffset(in, allOffsets[mainIFDIndex], nextPointer);
           saver.overwriteIFDValue(in, currentFullResolution, IFD.SUB_IFD, subIFDOffsets);
           saver.close();
+          in.close();
```

Comment:
I'd be for this being in a try-finally or try-resource block.

Reply:
The full block needs to be reviewed, I was trying to keep it "simple" but it is not going to work
I will get back to it